package.json

```diff
@@ -45,7 +45,7 @@
         "cookie-parser": "^1.4.4",
         "csurf": "^1.10.0",
         "express": "^4.16.4",
-        "govuk-frontend": "2.9.0",
+        "govuk-frontend": "3.10.0",
```

Comment:
Wonder why this doesn't use caret notation.

Reply:
They do seem to use patch versions, might be wise to change this next time we go into this file.